pom.xml

```diff
@@ -141,6 +141,17 @@
     <dependencies>
 
         <!-- Unit tests/mocking -->
+        <dependency>
+	    <groupId>org.openjdk.jmh</groupId>
+	    <artifactId>jmh-core</artifactId>
+	    <version>1.37</version>
```

Comment:
Add constant like in https://github.com/ponder-lab/Stream-Performance-Evaluation/blob/4c67246f6789e392d8a64b2d7d3da1d0555af035/performance/java-design-patterns/java-design-patterns.patch#L1151